sample.js

```diff
@@ -1,31 +1,29 @@
 var user = entity('User', {
+  username: field(String, {
```

Comment:
I think this file is not valid any longer... could you delete it?

Reply:
Yes